content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,355 @@
+
+The localization requires an Uruware account, which enables users to generate electronic documents
+within Odoo.
+
+Glossary
+--------
+
+The following terms are used throughout the Uruguayan localization:
+
+- **DGI**: *Dirección General Impositiva* is the government entity responsible for enforcing tax
+  payments in Uruguay.
+- **EDI**: *Electronic Data Interchange* refers to the sending of electronic documents.
+- **Uruware**: is the third-party organization that facilitates the interchange of electronic
+  documents between companies and the Uruguayan government.
+
+Configuration
+=============
+
+Modules installation
+--------------------
+
+:ref:`Install <general/install>` the following modules to get all the features of the Uruguayan
+localization:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Uruguay - Accounting`
+     - `l10n_uy`
+     - The default :doc:`fiscal localization package <../fiscal_localizations>`, adds accounting
+       characteristics for the Uruguayan localization, which represent the minimum configuration
+       required for a company to operate in Uruguay according to the guidelines set by the |DGI|.
+       The module's installation automatically loads: chart of accounts, taxes, documents types, and
+       tax supported types.
+   * - :guilabel:`Uruguay Accounting EDI`
+     - `l10n_uy_edi`
+     - Includes all the technical and functional requirements to generate and validate
+       :doc:`Electronics Documents <../accounting/customer_invoices/electronic_invoicing>`, based on
+       the technical documentation published by the |DGI|. The authorized documents are :ref:`listed
+       above <uruguay/intro>`.
+
+.. note::
+   Odoo automatically installs the base module *Uruguay - Accounting* when a database is installed
+   with `Uruguay` selected as the country. However, to enable electronic invoicing, the *Uruguay
+   Accounting EDI* (`l10n_uy_edi`) module needs to be manually :ref:`installed <general/install>`.
+
+Configure your company
+----------------------
+
+To configure your company information, go to the :menuselection:`Settings --> Companies: Update
+Info` and configure the following:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`
+
+- :guilabel:`Tax ID`: Enter the identification number for the selected taxpayer type.
+- :guilabel:`DGI Main Branch Code`: This is part of the XML when creating an electronic document. If
+  this field is not set, all electronic documents will be rejected.
+
+  To find the *DGI Main Branch Code*, follow these steps:
+
+  #. From your :ref:`Uruware account <l10n_uy/uruware-account>`, go to :menuselection:`Servicios en
+     línea DGI --> Registro único tributario --> Consulta de datos`.
```

Comment:
Hello @samueljlieber and @ewa-odoo 

Sorry, checking this part, what we're pointing is to the reference from the DGI (and it is correct)... But there are 2 options here, change the "ref: 'Uruware account' to DGI or, the instructions change them to the way they should find the Branch code in Uruware. Both are correct. My opinions is to the first (change to DGI) since at this point the Uruware account has not been created.

Let us know please.

Thanks!

fernando

@marianaluz

Reply:
Hey @samueljlieber 

This one was pending, as I can see

```suggestion
  #. From your :ref:`DGI account <https://servicios.dgi.gub.uy/serviciosenlinea>`, go to :menuselection:`Servicios en
     línea DGI --> Registro único tributario --> Consulta de datos`.
```

Not sure if this is the way to put in the DGI website, but you can add it the right way ;).

Cheers!

Fernando

Reply:
Hi @samueljlieber I don't see this change yet, could you please check? 😬